extended-tracer/build.gradle.kts

```diff
@@ -0,0 +1,13 @@
+plugins {
+  id("otel.java-conventions")
+  id("otel.publish-conventions")
```

Comment:
@zeitlinger we discussed in Java SIG today, and would like to merge this without publishing, and then move it into the core repo incubator

```suggestion
```